cli/popper/cli.py

```diff
@@ -58,7 +58,7 @@ def get_command(self, ctx, name):
             raise UsageError(
                 "Command '" + name + "' doesn't exist. " +
                 "\nType 'popper --help' for more."
-                + message
+                + message + '\n' + str(e)
```

Comment:
The str(e) is not required imo after the error has been determined

Reply:
@ankan17 - I think it is important that we keep this. Errors like ImportError and NameError both will fall under the UsageError and in those situations a simple `Command add doesn't exist` may not be that helpful. It will save our time while debugging if we know the problem exactly.